### PR TITLE
Fix start script not transpiling javascript

### PR DESCRIPTION
**Pull request by sygint**

### CODE REVIEW

**Review by SmolinPavel — APPROVED** [review]
That's the same solution I came up with 👍

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "lib/index.js",
   "scripts": {
     "build": "babel lib -d dist",
-    "start": "npm run build && nodemon dist/index.js",
+    "start": "npm run build && nodemon --exec babel-node index.js",
```
Comment (mengdage) [inline package.json]: we can remove the `npm run build` since we don't run the `dist/index.js`

**Comment by gnarmedia** [comment]
woops, fixing...

**Comment by gnarmedia** [comment]
It seems like this entire commit doesn't work and failed to include the README.md updates, so I was just thinking, would it just be better to do this with babel-register? that's how I do it locally and that's how the README explains doing it with mocha...

**Comment by gnarmedia** [comment]
ok, I think I finally got it all fixed including all of the necessary README updates.

**Comment by princenazi** [comment]
babel-node is too heavy for production

**Comment by gnarmedia** [comment]
> babel-node is too heavy for production

if babel-node is too production heavy, what about compiling and using the /dist directory again? I can revise it.

**Pull request closed without merging**